Source/MgsLib/LibDG.cpp

```diff
@@ -2667,41 +2667,28 @@ MGS_FUNC_IMPLEX(0x407122, LibGV_407122, LIBDG_IMPL);
 
 void CC OrderingTableAdd_4034C6(int pPrimDataStart, int count, int size)
 {
-    MGS_FORCE_ENOUGH_SPACE_FOR_A_DETOUR;
-
-    /*
-    DWORD* dword_991E40_1_ot_ptr = (DWORD*)gMatrix_dword_991E40[1];
-    const int dword_991E40_2_field_2E_w_or_h = gMatrix_dword_991E40[2];
-    if (count - 1 >= 0)
+    DWORD* dword_991E40_1_ot_ptr = (DWORD*)((DWORD*)gScratchPadMemory_991E40.field_0_raw.field_0)[1];
+    const DWORD dword_991E40_2_field_2E_w_or_h = ((DWORD*)gScratchPadMemory_991E40.field_0_raw.field_0)[2];
+    BYTE* pData = (BYTE*)pPrimDataStart;
+    for (int i = 0; i < count; ++i, pData += size)
     {
-        int pData = pPrimDataStart;
-        do
-        {
-            const int tag = *(WORD *)pData;
-            if (tag > 0)
-            {
-                signed int v6 = tag - dword_991E40_2_field_2E_w_or_h;
-                if (v6 < 0)
-                {
-                    v6 = 0;
-                }
+        const DWORD value0 = *(WORD*)pData;

```

Comment:
This seems to be the first word of a prims tag value. Not sure what it means or how it used but might be better to call it as such. It gets used in the rank actor this way too, see `PolyTag` here: https://github.com/paulsapps/msgi/blob/master/Source/MgsLib/Actor_Rank.cpp#L103